### PR TITLE
Validation script

### LINKED ISSUE #789: Code validation script
**Is your feature request related to a problem? Please describe.**
We should have an interactive validation script, that checks current codebase, shows warnings/errors and helps developer to fix them.

**Describe the solution you'd like**
The interactive script should run the same validators travis runs:
1. It should update/install eslint, flake8, isort, coverage, pipdeptree based on a dedicated requirements.txt file.
2. It should show pipdeptree warnings
3. It should show eslint warnings. It should separately show all warnings in files, modified on current branch.
4. It should show flake8 warnings. It should separately show all warnings in files, modified on current branch. 
5. It should show isort warnings and automatically prompt you to sort imports on files with warnings.
6. It should run all UT's.

### PR DESCRIPTION
**Pull request by VakarisZ**
# What does this PR do? 

Fixes #789 

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [x] Is the TravisCI build passing? 

## Testing Checklist

* [x] Have you successfully tested your changes locally? Elaborate:
    > Tested by running validation script locally

### CODE REVIEW

File: ci_scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# About
+
+Run this script to validate your code locally and auto fix/format the problems before pushing.
+
+# Usage
+
+You've got to manually download swimm for swimm validation.
+run from `infection_monkey` directory: `powershell .\ci_scripts\validate.ps1`
```
Comment (acepace) [inline ci_scripts/README.md:8]: No linux support?

Comment (VakarisZ) [reply ci_scripts/README.md:8]: Correct

**Comment by acepace** [comment]
@VakarisZ @itaymmguardicore still relevant if we won't stay in travisCI due to price change?

**Comment by VakarisZ** [comment]
We're not migrating yet. Lets merge.

**Comment by acepace** [comment]
Go for it

**Review by supkisids50 — COMMENTED** [review]
O

**Pull request merged**